README.md

```diff
@@ -3,13 +3,13 @@
 > Plays everywhere, every time.

```

Comment:
Update the README.md with usage for the current `npm script` based build system.